modules/material/themes/material/core/loginuserpass.php

```diff
@@ -82,16 +83,18 @@ function onRecaptchaLoad() {
                         <?php
                         $username = htmlentities($this->data['username'] ?? null);
                         ?>
-                        <input type="text" required name="username" class="mdl-textfield__input mdl-color-text--black" value="<?= $username ?>"
-                               <?= empty($username) ? 'autofocus' : '' ?> id="username"/>
+                        <input type="text" required name="username" class="mdl-textfield__input mdl-color-text--black"
+                               value="<?= $username ?>"
```

Comment:
I would have swapped the order of class and value so line wrapping would be nicer.